src/components/SpectralProfiler/SpectralProfilerSettingsPanelComponent/SpectralProfilerSettingsPanelComponent.tsx

```diff
@@ -72,6 +77,14 @@ export class SpectralProfilerSettingsPanelComponent extends React.Component<Widg
         });
     }
 
+    @action setCursorInfo(state: boolean) {
```

Comment:
remove unused functions `setCursorInfo` and `handleSecondaryAxisCursorInfoChanged`